### PR TITLE
ci: install as simple application

### LINKED ISSUE #700: e2e: Check upgrade happens without downtime
Follow-up of https://github.com/rancher/elemental/issues/651

We should verify the upgrade happens without down time as fleet should orchestrate the upgrade to make it possible.
The idea is to deploy an application in the cluster and check its availability while upgrading the whole cluster.

### PR DESCRIPTION
**Pull request by ldevulder**
Install a simple application and check it in different upgrade scenari.

NOTE: the test only runs on K3s for now, as there is no load-balancer installed by default on RKE2.

Should fix #700.

Verification runs:
- [CLI-K3s-OBS_Dev - with Hardened](https://github.com/rancher/elemental/actions/runs/5509110071)
- [CLI-K3s-OS-Upgrade](https://github.com/rancher/elemental/actions/runs/5508396251)

Regression tests, RKE2 shouldn't be impacted:
- [CLI-RKE2-OBS_Dev](https://github.com/rancher/elemental/actions/runs/5509125892)
- [CLI-RKE2-OS-Upgrade](https://github.com/rancher/elemental/actions/runs/5509137287)
- [UI-K3s-OBS_Dev](https://github.com/rancher/elemental/actions/runs/5509158876)

### CODE REVIEW

**Review by kkaempf — APPROVED** [review]
lgtm

File: .github/workflows/cli-rke2-obs_staging.yaml [hunk 1]
```diff
@@ -28,6 +31,7 @@ jobs:
       test_description: "Manual - CLI - Parallel - Deployment test with Standard RKE2"
       ca_type: private
       cluster_name: cluster-rke2
+      luster_type: ${{ inputs.cluster_type }}
```
Comment (juadk) [inline .github/workflows/cli-rke2-obs_staging.yaml]:
```suggestion
      cluster_type: ${{ inputs.cluster_type }}
```

**Review by juadk — CHANGES_REQUESTED** [review]
LGTM, only a small typo to change.

File: .github/workflows/cli-rke2-obs_staging.yaml (hunk 1 above)
Comment (ldevulder) [reply .github/workflows/cli-rke2-obs_staging.yaml]: Fixed, thanks!

**Review by juadk — APPROVED** [review]

**Pull request merged**